theolive/platform/security/token-based-security.mdx

```diff
@@ -12,15 +12,24 @@ In this document, we will
 
 When enabling token security on a channel alias, we expect you to share the shared (private) key in case of HS256/HS512 or the public key in case of RS256/RS512 encryption. This will be used on CDN level to determine if a request (with a Bearer token passed in the `Authorization` header) is valid or not. It's up to you to configure this header correctly through our [npm player](https://www.npmjs.com/package/theoplayer). Later in this document we will explain how.
 
-In the bearer token that gets sent to us, we expect the following properties to be available:
+In the bearer token that gets sent to us, we _require_ the following properties to be provided in the `payload`:
 
 - `exp`: date in epoch format until which the JWT token is valid
 - `nbf`: optional date in epoch format. Stands for "not before" and acts as a "start date" of the JWT to be valid.
```

Comment:
You mention it's optional here and elsewhere but a few lines above you say we require the fields so that implies it's non optional.